webpack.config.js

```diff
@@ -103,6 +104,7 @@ const config = {
 			minChunks: (module) => module.context && module.context.indexOf("node_modules") !== -1,
 		}),
 	],
+	stats: "minimal",
```

Comment:
Same comment as #1943 re: `npm run build` and `npm run watch`, they're now pretty much silent.